src/txmempool.cpp

```diff
@@ -612,12 +612,18 @@ CTxMemPool::~CTxMemPool() {}
 
 CCustomCSView &CTxMemPool::accountsView() {
     if (!acview) {
+        LOCK(cs_main);
```

Comment:
Could this potentially deadlock the node?

Reply:
cs_main is already held there. I'll remove the lock for this function and resetAccountsView() as they are redundant.